src/HttpRepository/AirtableHttpRepository.php

```diff
@@ -2,30 +2,22 @@
 
 declare(strict_types=1);
 
-namespace Yoanbernabeu\AirtableClientBundle;
+namespace Yoanbernabeu\AirtableClientBundle\HttpRepository;
 
 use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
 use Symfony\Contracts\HttpClient\HttpClientInterface;
-use Symfony\Contracts\HttpClient\ResponseInterface;
+use Yoanbernabeu\AirtableClientBundle\AirtableRecord;
+use Yoanbernabeu\AirtableClientBundle\AirtableRepositoryInterface;
 
-/**
- * AirtableClient.
```

Comment:
This kind of comments don't any plus value, 

:bulb: add comment only if the code is not enough explicit